.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -36,3 +36,12 @@ _What is the actual result of the above steps?_
 ## 📷 Screenshots
 
 _Are there any useful screenshots? WinKey+Shift+S and then just paste them directly into the form_
+
+## 📝 Log files
+
+_Are there any useful log files. If yes upload the file in here_
+
+<details>
```

Comment:
this needs crisper guidance.  No one would know about the log files unless we told them where to look.